docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3'
```

Comment:
Remove this file, please, and add later if it's necessary in other PR. 

Note: I would like to do unittest without bbdd services.

Reply:
done